feature/gnmi/otg_tests/telemetry_fan_tray_type_test/README.md

```diff
@@ -0,0 +1,30 @@
+# gNMI-??: Telemetry: Fan Tray Type Test
```

Comment:
@dplore do you know where I find what gNMI version to use here?

Reply:
You can select an unused number from [testregistry.textproto](https://github.com/openconfig/featureprofiles/blob/main/testregistry.textproto)